account_mass_reconcile/models/mass_reconcile.py

```diff
@@ -166,7 +166,7 @@ def _last_history(self):
     last_history = fields.Many2one(
         'mass.reconcile.history',
         string='Last history', readonly=True,
-        compute='_last_history',
+        compute='_compute_last_history',
```

Comment:
These changes of method names should be avoided in published versions as it could break existing installations.

At the very least, `_last_history = _compute_last_history` should be added somewhere for compatibility if the change is needed.